### ISSUE TITLE
Error when displaying server online 

### ISSUE THREAD

**Issue by TonyBombony** [RahulR100/mcstatusbot#52]
Since yesterday the bot periodically ~every hour began to show that the server is offline and has 0 people on it, although it is not true
After a while, of course it recovers, but after some more time it breaks again and it's not pleasant :(
![Screenshot_3](https://github.com/RahulR100/mcstatusbot/assets/154249719/ac40a128-8e73-4955-8eaa-fe14d8aa9e5d)
![Screenshot_4](https://github.com/RahulR100/mcstatusbot/assets/154249719/b6f4cb33-3a16-4b2d-b7eb-3450440bff7c)

**Comment by RahulR100** [comment]
Yup this seems to be a cache issue on our end we are looking into this!

**Comment by RahulR100** [comment]
A fix has been applied and this issue should no longer exist. Please let us know if you see this issue again.
